src/Directory.Packages.props

```diff
@@ -6,8 +6,8 @@
 
     <!-- Install from version 5.0.11 included all the way up to 8.x.x, but don't go up a major version -->
     <PostgresLowVersion>5.0.11</PostgresLowVersion>
-    <PostgresHighVersion>8</PostgresHighVersion>
-    <PostgresVersion>[$(PostgresLowVersion), $(PostgresHighVersion))</PostgresVersion>
+    <PostgresHighVersion>9</PostgresHighVersion>
+    <PostgresVersion>[$(PostgresLowVersion), $(PostgresHighVersion)]</PostgresVersion>
```

Comment:
The square brackets weren't closed properly since the commit of @Arkatufus on 10/4/2023